packages/hash/api/src/graphql/resolvers/knowledge/user/me.ts

```diff
@@ -9,11 +9,10 @@ export const me: ResolverFn<
   QueryMeArgs
```

Comment:
The `*QueryVariables` come from the queries defined in the shared library (`../shared/src/queries/**/*.ts` in codegen.yml for API). If we want to make use of those, we have to add a standalone query (not extending the `Query` type). Although this would be consistent with the frontend, it would mean changing a lot of queries in the backend to get rid of `Args`.

Reply:
Nevermind I think we've finally figured it out 😁 
<img width="520" alt="image" src="https://user-images.githubusercontent.com/25749103/203525687-16bbedc2-3774-4d65-b680-787d5bc79a86.png">
<img width="670" alt="image" src="https://user-images.githubusercontent.com/25749103/203525874-63e7998f-6f51-4c05-a2b5-6a3c178aeb6c.png">
<img width="679" alt="image" src="https://user-images.githubusercontent.com/25749103/203525910-843bf8e6-6900-4a35-95fc-5946ae52399f.png">

_Resolvers_ should use the `*Args` types, and the `useLazyQuery` and `useMutation` (etc.) functions should use `*Variables`

Really would be nice if there was actual type safety on this